### ISSUE TITLE
'dict object' has no attribute 'legacy_list_objects'

### ISSUE THREAD

**Issue by harenber** [hadret/ansible-role-restic#10]
Hi,

with the new version 1.3.0, I had to define `legacy_list_objects` in restic_repos, otherwise I get a 

```
"msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'legacy_list_objects'
```

in the `deploy wrapper scripts`  task when deploying the Playbook. 

Thanks for your role. Very useful!!

**Comment by hadret** [comment]
Hey, sorry for trouble and thank you for the report. This should be fixed in tagged release [v1.3.3](https://github.com/hadret/ansible-role-restic/releases/tag/v1.3.3) 👍🏻

**Comment by harenber** [comment]
Thank you :)
